10x/hca-dcp/submit_10x_count.wdl

```diff
@@ -0,0 +1,37 @@
+task submit {
+  String i
+
+  command {
+    cat ${i} > out.txt
```

Comment:
Does `${i}` refer to a local file?

Reply:
Hmm, I think this is just a copy and paste from the placeholder submission WDL that I wrote for SS2, so this is my bad. Once this PR is merged we'll immediately replace this with a real submission WDL from this other PR:
https://github.com/HumanCellAtlas/skylab/blob/409ddb7501a74e4e86391fe4c7fb4fc70d19a45c/hca-dcp/submit.wdl
You're right though, we probably want to do echo instead of cat in this placeholder WDL.

Reply:
Yes! Thanks for pointing that out @mckinsel